### PR TITLE
[Testmerge Ready] Ancient Station - Remake

### PR DESCRIPTION
**Pull request by ForrestWick**
## About The Pull Request
_In the first years of Nanotrasen's rise to economic power, the advent of Plasma was being first discovered. Initially regarded as a fuel source, Nanotrasen began to amass science teams to experiment with the strange element on secretive research stations hidden outside the borders of SolGov controlled space. There were initially a handful of these stations, each one housing a small group of individuals kept silent with NDA contracts and significantly large paychecks. For unknown reasons however, all three were decommissioned and the crews pulled from their respective stations. Except for one._

_200 years later, what remains of the crew of the NTSS002, Designation DCB0201, wakes up from cryo stasis after the station AI warned and ushered them to the cryo pods, assuring them they would awaken in 8 hours once a radiation storm had passed. These men and women awake to find their station a rusted derelict, most of it in tatters, and an unknown enemy stalking the halls of the once tightly-locked science wing. They must work together to survive, and find out just what happened aboard their station. Or die trying..._

Preview Images:
https://i.imgur.com/mUprEDM.png
https://i.imgur.com/fCR936Z.png
https://i.imgur.com/FVooMLT.png
https://i.imgur.com/7LaeeTo.png
https://i.imgur.com/TNc53aY.png
https://i.imgur.com/ZHr3ro0.png
https://i.imgur.com/nultkln.png

## Why It's Good For The Game
Old Station/Charlie Station originally got a small rework from me some months ago, adding new areas and fixing a few of the old ones. This, however, is a new, complete overhaul of the ruin, with new content, a new weapon, a revamped layout, new areas, and even a special gift meant for the Old Station Miner in particular.

This PR bumps the spawns up to 8, adding an old station Atmos Tech on top of the previous additions. The ruin/ghost role offers new and returning players a new experience with a much better layout than before. This ruin will allow you to practice combat, construction, engineering, atmos, medical, space mining, botany/cooking with limited resources, and ports over the Staff Assistant jumpsuit from Citadel, initially ported from Goon. Players who spawn into the role must work together (yes that means roleplay) to reclaim the station from hostile occupants, repair the station's power grid and atmospherics, and find out just what happened 200 years ago.

## Changelog
:cl:
add: Completely new oldstation ruin/ghost role map
add: staff assistant jumpsuit added to clothesmate and old station assistant loadout
tweak: old station chaplain no longer has their holy status, due to my own coding knowledge limitations. If someone else is willing to help fix this, let me know or open a PR on your own later.
config: added oldstationnew.dmm to the ruins blacklist
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Add newlines to file please.

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -109,7 +109,8 @@
 		            /obj/item/clothing/suit/apron/purple_bartender = 2,
 		            /obj/item/clothing/under/rank/civilian/bartender/purple = 2,
 		            /obj/item/clothing/suit/toggle/suspenders/blue = 2,
-		            /obj/item/clothing/suit/toggle/suspenders/gray = 2)
+		            /obj/item/clothing/suit/toggle/suspenders/gray = 2,
+		            /obj/item/clothing/under/staffassistant = 5) //Skyrat edit, ported from Cit/Goon
```
Comment (Gandalf2k15) [inline code/modules/vending/clothesmate.dm]: Unrelated PR change, please add this to an existing module.

File: config/spaceruinblacklist.txt [hunk 2]
```diff
@@ -48,4 +48,5 @@
 #_maps/RandomRuins/SpaceRuins/whiteshipdock.dmm
 #_maps/RandomRuins/SpaceRuins/whiteshipruin_box.dmm
 #_maps/RandomRuins/SpaceRuins/forgottenship.dmm
+#modular_skyrat/master_files/_maps/randomruins/spaceruins/oldstationnew
```
Comment (Gandalf2k15) [inline config/spaceruinblacklist.txt:51]: will require testing and a config update.

File: modular_skyrat/master_files/code/game/area/areas/ruins/space.dm [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/area/ruin/space/has_grav/ancientstation/xenobio
```
Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/game/area/areas/ruins/space.dm]: Please make this a module.

File: modular_skyrat/master_files/code/game/objects/items/cards_ids.dm [hunk 4]
```diff
@@ -0,0 +1,59 @@
+//charliestation stuff
```
Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/game/objects/items/cards_ids.dm]: Please make this part of the same module.

File: modular_skyrat/master_files/code/game/objects/structures/ghost_role_spawners.dm [hunk 5]
```diff
@@ -0,0 +1,132 @@
+/obj/effect/mob_spawn/human/oldmine
```
Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/game/objects/structures/ghost_role_spawners.dm]: Again, needs to be in a module.

File: modular_skyrat/master_files/code/modules/clothing/miscellaneous.dm [hunk 6]
```diff
@@ -0,0 +1,7 @@
+/obj/item/clothing/under/staffassistant
```
Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/modules/clothing/miscellaneous.dm]: Same here, module.

File: modular_skyrat/master_files/code/modules/ruins/spaceruin_code/oldstation.dm [hunk 7]
```diff
@@ -0,0 +1,69 @@
+///////////	Oldstation items
```
Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/modules/ruins/spaceruin_code/oldstation.dm]: Module it.

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
To make this a module, go ahead and create a new folder under "modules" and then name it to something like "oldstation" this will be your module ID. Then put all these files into that folder accordingly. So it should look like modular_skyrat\modules\oldstation\code... etc. Also include a readme.md which can be found by looking at any of the other modules, and fill it out accordingly.

File: modular_skyrat/master_files/code/game/area/areas/ruins/space.dm (hunk 3 above)
Comment (ForrestWick) [reply modular_skyrat/master_files/code/game/area/areas/ruins/space.dm:1]: What do you mean?
